src/gpt-auto-generator/gpt-auto-generator.c

```diff
@@ -453,33 +453,37 @@ static int add_boot(const char *what) {
         _cleanup_blkid_free_probe_ blkid_probe b = NULL;
         const char *fstype = NULL, *uuid = NULL;
         sd_id128_t id, type_id;
+        const char *esp;
         int r;
 
         assert(what);
 
         if (!is_efi_boot()) {
```

Comment:
Hm, do we really want to skip mounting the ESP in this case? It seems reasonable to mount it, even if we boot from BIOS. Not related to this PR of course.